rpc_interface/rpc_interface.idl

```diff
@@ -66,4 +66,6 @@ import "wtypes.idl";
             [ in, ref ] ebpf_program_load_info * info,
             [ out, ref ] uint32_t * logs_size,
             [ out, size_is(, *logs_size), ref ] char** logs);
+
+        ebpf_result_t verify_and_authorize_native_image([ in, string ] char* image_path);
```

Comment:
Consider adding
`cpp_quote( "#if (defined(CONFIG_BPF_JIT_DISABLED) && defined(CONFIG_BPF_INTERPRETER_DISABLED)))`

Reply:
Thank you for the suggestion, but this doesn't appear to do what is expected. The client and server stubs still be generated for verify_and_load_program. All this will do is control what is emitted in the generated header file.